### PR TITLE
[CallWithChat] Left Align Participant pane items in desktop and mobile view

### PR DESCRIPTION
**Pull request by edwardlee-msft**
# What
Left align items in people pane/sidepane in desktop and mobile view

# Why
Many items in the people pane in mobile and desktop were not aligned
https://skype.visualstudio.com/SPOOL/_workitems/edit/2632003/

# How Tested

# Process & policy checklist
Tested locally on desktop and mobile view

### CODE REVIEW

**Review by prprabhu-ms — COMMENTED** [review]
The context menu in desktop also got extra margins. Don't think that is intended: 
image

**Comment by dmceachernmsft** [comment]
should you have the extra space between the participants in the CallWithChatComposite? 
image

**Comment by edwardlee-msft** [comment]
> should you have the extra space between the participants in the CallWithChatComposite? image

Yes, I believe so. 
When looking at the figma, there should actually be space between participants in mobile view, at least in CallWithChat. (But that is out of scope for this PR).

**Review by dmceachernmsft — APPROVED** [review]

File: packages/react-components/src/components/styles/ParticipantList.styles.ts [hunk 1]
```diff
@@ -8,8 +8,8 @@ import { ParticipantListItemStyles } from '../ParticipantList';
  * @private
```
Comment (anjulgarg) [inline packages/react-components/src/components/styles/ParticipantList.styles.ts:8]: Unnecessary change?
Just flipping the order?

**Review by anjulgarg — APPROVED** [review]
